README.md

```diff
@@ -176,6 +176,24 @@ format:
 <details open>
 <summary>Toggle</summary>
 
+- [Policy Rehearsing: Training Generalizable Policies for Reinforcement Learning](https://openreview.net/forum?id=m3xVPaZp6Z)
+  - Chengxing Jia, Chenxiao Gao, Hao Yin, Fuxiang Zhang, Xiong-Hui Chen, Tian Xu, Lei Yuan, Zongzhang Zhang, Zhi-Hua Zhou, Yang Yu
+  - Key: Reinforcement Learning, Model-based Reinforcement Learning, Offline Reinforcement Learning
+  - OpenReview: 8, 8, 8, 6
+  - ExpEnv: [d4rl](https://github.com/rail-berkeley/d4rl)
+
+- [Efficient Dynamics Modeling in Interactive Environments with Koopman Theory](https://link.zhihu.com/?target=https%3A//openreview.net/forum%3Fid%3DfkrYDQaHOJ)
```

Comment:
The redirection of zhihu should be removed from this link.

Reply:
Cheers!